app/src/main/AndroidManifest.xml

```diff
@@ -57,6 +57,24 @@
             android:label="@string/app_name"
             android:screenOrientation="portrait"/>
 
+        <activity
+            android:name=".reminders.ReminderActivity"
+            android:label="@string/app_name"
+            android:screenOrientation="portrait"/>
+
+        <!-- These control local reminder notification scheduling and display -->
+        <receiver android:name="org.sagebionetworks.research.sageresearch.reminders.MpReminderAlarmReceiver" />
```

Comment:
Does this need an intent filter?

Reply:
Not that I know of.  What filters would you put on it?